src/main/java/Duke.java

```diff
@@ -1,10 +1,350 @@
+import java.util.Locale;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
 public class Duke {
+
+    private static void appendToFile(String filePath, String textToAppend) throws IOException {
+        FileWriter fw = new FileWriter(filePath, true); // create a FileWriter in append mode
```

Comment:
Should comments be indented relative to their position in the code?
```suggestion
        // create a FileWriter in append mode
        FileWriter fw = new FileWriter(filePath, true); 
```

Reply:
you are right.